playwright.config.ts

```diff
@@ -10,19 +10,21 @@ import * as dotenv from 'dotenv';
  */
 dotenv.config();
 
+const TIMEOUT_TIME = 60_000;
+
 /**
  * See https://playwright.dev/docs/test-configuration.
  */
 const config: PlaywrightTestConfig = {
   testDir: './tests',
   /* Maximum time one test can run for. */
-  timeout: 30 * 1000,
+  timeout: TIMEOUT_TIME,
   expect: {
     /**
      * Maximum time expect() should wait for the condition to be met.
      * For example in `await expect(locator).toHaveText();`
      */
-    timeout: 5000,
+    timeout: 10000,
```

Comment:
I'd pull up a constant to near line 13. Then rename TIMEOUT_TIME and this one such that they're easier to quickly understand at a glance. Maybe MAXIMUM_TIME_PER_TEST_IN_MILLISECONDS and MAXIMUM_TIME_TO_WAIT_FOR_CONDITION_IN_MILLISECONDS, although I think even this needs comments. 

Is it the case that a test where the sum of the `expect.timeout`s > `timeout` will bail / quit at the root timeout?

Reply:
Nice suggestion! Updated!